README.rst

```diff
@@ -118,6 +121,80 @@ Short instructions
 
    -  Admin - Username: admin \| Password: admin
 
+2. Yaksh Installation as a django-app
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+1. pip install git+https://github.com/FOSSEE/online_test
+
+2. Add following apps to the INSTALLED_APPS in the project settings:
+    - yaksh
+    - taggit
+    - grades
+    - social.apps.django_app.default
+
+3. On terminal, run the migrate command:
+
+   ::
+
+       $ python manage.py migrate
+
+4. Add following in the project urls.py
+
+   ::
+
+        from django.conf.urls import url, include
+
+        urlpatterns += [
+            url(r'^exam/', include('yaksh.urls', namespace='yaksh', app_name='yaksh')),
+            url(r'^exam/reset/', include('yaksh.urls_password_reset')),
+            url(r'^', include('social.apps.django_app.urls', namespace='social')),
+            url(r'^grades/', include('grades.urls', namespace='grades', app_name='grades')),
+        ]
+
+Note, here there should be static url for Media files,
+this is necessary for file upload/download to work. Following is for reference:
+
+   ::
+
+        In project settings file
+
+        MEDIA_URL = '/'
```

Comment:
If I keep` MEDIA_URL` as `/` I am unable to attempt the quiz. It gives me 404. If I change it to let's say `/data/` then it works I don't know if this is a problem with my system. But if this works. then fine.

Reply:
That is just a reference.
```/``` should work. You may have put data in media url or media root.
Please check.

Reply:
I followed whatever is mentioned in the README. It failed for me. Ok, if that works.